lib/widgets/contextual_close_action.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:contextualactionbar/actions/action_mode.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../controller/items_controller.dart';
+import '../typedef/handle_items.dart';
+
+class ContextualCloseAction<T> extends StatelessWidget {
```

Comment:
What's the usecase for this ContextualCloseAction?. The material design guideline doesn't have this specification. Personally i will prefer a callback to handle this or better still use 
ContextualAction instead of create an entirely new type of widget.

Reply:
So initially I tried to just pass `HandleItems<T>`, but the type was getting erased (sorry, not an expert in Dart but that appears to be what was happening). Wrapping it in `ContextualAction<T>` got rid of that issue, but if you know of a way to prevent the type from being erased, definitely would like to go that route. The use case for this is simply to do an action on the list of selected items when the close button is selected, just like a `ContextualAction` does.